ui/src/app/cron-workflows/components/cron-workflow-list/cron-workflow-list.tsx

```diff
@@ -152,19 +152,35 @@ export function CronWorkflowList({match, location, history}: RouteComponentProps
                                         </div>
                                         <div className='columns small-2'>{w.metadata.namespace}</div>
                                         <div className='columns small-1'>{w.spec.timezone}</div>
-                                        <div className='columns small-1'>{w.spec.schedule}</div>
+                                        <div className='columns small-1'>
+                                            {w.spec.schedule != ''
+                                                ? w.spec.schedule
+                                                : w.spec.schedules.map(schedule => (
+                                                      <>
+                                                          {schedule}
+                                                          <br />
+                                                      </>
+                                                  ))}
+                                        </div>
                                         <div className='columns small-2'>
-                                            <PrettySchedule schedule={w.spec.schedule} />
+                                            {w.spec.schedule != '' ? (
+                                                <PrettySchedule schedule={w.spec.schedule} />
+                                            ) : (
+                                                <>
+                                                    {w.spec.schedules.map(schedule => (
+                                                        <>
+                                                            <PrettySchedule schedule={schedule} />
+                                                            <br />
+                                                        </>
```

Comment:
I'm pretty sure that most of these instances can be simplified with a `<p>` paragraph element instead of a `<>` fragment + a `<br />` line break

```suggestion
                                                        <p><PrettySchedule schedule={schedule} /></p>
```

Reply:
Also how does this look when `schedules` only contains a single schedule? It seems like there may be an unnecessary new line in that case since it is currently treated differently from `schedule`

Reply:
I tested this suggestion and it seems that replacing `<> <br /> </>` with:
```
  <p>
      <PrettySchedule schedule={schedule} />
  </p>
```
 seems to be adding an unnecessary new line after the last schedule, as you can see here:

<img width="1182" alt="image" src="https://github.com/argoproj/argo-workflows/assets/22349222/7c650790-9536-4dbe-9ed1-1ba7e82d031a">

while the current behavior is shown as:

<img width="1186" alt="image" src="https://github.com/argoproj/argo-workflows/assets/22349222/f0634968-8b36-4a67-8c96-407de819b3da">

no new line

Reply:
Hmm we use `<p>` elements in the [new markdown title & description feature](https://github.com/argoproj/argo-workflows/pull/12580/files#diff-b9b22a6a512902e9b276dae962c848ad6b87d55b8a4daf66252e9524957274e4R9). Might need to add [similar CSS](https://github.com/argoproj/argo-workflows/pull/12580/files#diff-ab3fd4b11164b06bb926ee013f016665956ebb2d569433d57b5ad177ff3d3470R3) here (`line-height`, `display`, and `vertical-align` in particular). That and it might not work properly if one column uses `<p>` while the other uses `<br />`.

Try that; if it doesn't work, I can take a look on my side

Reply:
Also were you able to check `schedules` with only one schedule as I mentioned [above](https://github.com/argoproj/argo-workflows/pull/12616#discussion_r1495318106)?